azure-kusto-ingest/azure/kusto/ingest/ingestion_properties.py

```diff
@@ -163,7 +163,7 @@ def __init__(
         dataFormat = kwargs.get("dataFormat", None)
         ingestionMapping = kwargs.get("ingestionMapping", None)
         ingestionMappingType = kwargs.get("ingestionMappingType", None)
-        ingestionMappingReference = kwargs.get("ingestionMappingType", None)
+        ingestionMappingReference = kwargs.get("ingestionMappingReference", None)
```

Comment:
Isn't this a breaking change even if the old value was incorrect?

Reply:
Why would it be a breaking change? Its a bug in backward compatibility code..

Reply:
Oh, thought this was just a name change, no I see the same property name was used twice